fullscreen.bs

```diff
@@ -178,7 +184,14 @@ enum FullscreenNavigationUI {
   "hide"
 };
 
+enum FullscreenKeyboardLock {
+  "application",
```

Comment:
It's probably worth renaming it to `browser`. [WebKit's implementation](https://github.com/WebKit/WebKit/pull/21374/files#diff-8b54e72a7adef28afc726456ac23fc80c68f4d870a1a37ddadf9ad5e22df1107R38) of the [original proposal](https://docs.google.com/document/d/191YzTVJRg4mIkAYCHYbDYFugO4xiLw6xE_lDuUOfItQ/edit?tab=t.0#) has uses a `browser` there, so it would be great to avoid confusion.